.github/workflows/merge-to-protected.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
+      - name: Install
+        run: npm install
+      - name: Build
+        run: npm run-script build
```

Comment:
OK, although not sure if after migration to vite it still holds but we will see I guess after merge